src/Lotto.js

```diff
@@ -1,18 +1,154 @@
+import { LOTTO_MESSAGES, LOTTO_NUMBER, LOTTO_PRIZE_MONEY } from "./constants/lotto.js";
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class Lotto {
-  #numbers;
+  #lottoNumbersStore = [];
+  #prizeMoney = 0;
+  #lottoResult = new Map([
+    [3, 0],
+    [4, 0],
+    [5, 0],
+    ["5B", 0],
+    [6, 0],
+  ]);
+
+  // 유효성 검사
+  #isLottoPriceValidate(price) {
+    if (price % LOTTO_NUMBER.DIVISION_PRICE === 0) {
+      return false;
+    }
+
+    return true;
+  }
+
+  #isLottoNumberLengthValidate(numbers) {
+    if (numbers.length === LOTTO_NUMBER.LENGTH) {
+      return false;
+    }
+
+    return true;
+  }
+
+  #isLottoNumberValidate(number) {
+    return LOTTO_NUMBER.MIN > number || number > LOTTO_NUMBER.MAX;
+  }
+
+  #isDuplicateLottoNumberValidate(numbers) {
+    return new Set(numbers).size !== LOTTO_NUMBER.LENGTH;
+  }
+
+  // 사용자 입력
+  async getUserInputLottoPrice() {
+    const userInput = await MissionUtils.Console.readLineAsync(LOTTO_MESSAGES.INPUT_LOTTO_PRICE);
+    const price = Number(userInput);
+
+    if (this.#isLottoPriceValidate(price)) {
+      throw new Error(LOTTO_MESSAGES.INVALID_LOTTO_PRICE);
+    }
+
+    return price;
+  }
+
+  async getUserInputLottoNumber() {
+    const userInput = await MissionUtils.Console.readLineAsync(LOTTO_MESSAGES.INPUT_LOTTO_NUMBER);
+    const userLottoNumbers = userInput.split(",").map((number) => Number(number));
+
+    if (this.#isLottoNumberLengthValidate(userLottoNumbers)) {
+      throw new Error(LOTTO_MESSAGES.SIX_LENGTH_LOTTO_NUMBER);
+    }
+
+    if (userLottoNumbers.some((number) => this.#isLottoNumberValidate(number))) {
+      throw new Error(LOTTO_MESSAGES.INVALID_LOTTO_NUMBER);
+    }
+
+    if (this.#isDuplicateLottoNumberValidate(userLottoNumbers)) {
+      throw new Error(LOTTO_MESSAGES.DUPLICATE_LOTTO_NUMBER);
+    }
+
+    return userLottoNumbers;
+  }
+
+  async getUserInputBonusNumber(userLottoNumbers) {
+    const userInput = await MissionUtils.Console.readLineAsync(LOTTO_MESSAGES.INPUT_BONUS_NUMBER);
+    const bonusNumber = Number(userInput);
+
+    if (this.#isLottoNumberValidate(bonusNumber)) {
+      throw new Error(LOTTO_MESSAGES.INVALID_LOTTO_NUMBER);
+    }
+
+    if (userLottoNumbers.includes(bonusNumber)) {
+      throw new Error(LOTTO_MESSAGES.DUPLICATE_LOTTO_NUMBER);
+    }
+
+    return bonusNumber;
+  }
+
+  // 로또 번호 생성, 리턴, 출력
+  #generateLottoNumbers() {
+    const lottoNumbers = MissionUtils.Random.pickUniqueNumbersInRange(LOTTO_NUMBER.MIN, LOTTO_NUMBER.MAX, 6);
```

Comment:
여기 파라미터로 전달된 6도 상수를 활용하면 좋을 것 같아요!
```suggestion
    const lottoNumbers = MissionUtils.Random.pickUniqueNumbersInRange(LOTTO_NUMBER.MIN, LOTTO_NUMBER.MAX, LOTTO_NUMBER.LENGTH);
```

Reply:
좋은 것 같습니다!🙃